### PR TITLE
Add missing tables and test for model table existence

### PR DESCRIPTION
**Pull request by notbakaneko**

### CODE REVIEW

File: tests/TableTest.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            $class::first();
+        } catch (Exception $e) {
+            return $class;
+        }
+    }
+
+    private function checkDir(string $basePath, string $namespace = '')
+    {
+        $errors = [];
+        $entries = scandir($basePath);
```
Comment (nanaya) [inline tests/TableTest.php]: There's this `File::allFiles()`.

File: tests/TableTest.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+class TableTest extends TestCase
+{
+    public function testTableExistence()
+    {
+        $this->modelsPath = app_path().'/Models';
+        $errors = $this->checkDir($this->modelsPath);
+
+        // prints a diff with the classes that errored.
+        $this->assertEquals([], $errors);
```
Comment (nanaya) [inline tests/TableTest.php]: Probably should use `assertSame`.

File: tests/TableTest.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+class TableTest extends TestCase
+{
+    private $modelsPath;
+
+    public function testTableExistence()
+    {
+        $errors = [];
+
+        $files = (new Finder)->in(app_path().'/Models')->files();
```
Comment (nanaya) [inline tests/TableTest.php]: Why don't just use the `File` thingy from laravel? ?_?

Comment (notbakaneko) [reply tests/TableTest.php]: It's a wrapper around `Finder` anyway ┐(ツ)┌

File: composer.json [hunk 4]
```diff
@@ -37,6 +37,7 @@
     "paypal/rest-api-sdk-php": "*",
     "predis/predis": "~1.0",
     "sentry/sentry-laravel": "*",
+    "symfony/finder": "*",
```
Comment (nanaya) [inline composer.json]: this probably isn't needed. If laravel changes the return, it'll explode in different way anyway and this wouldn't help ┐(°～° )┌

**Pull request merged**